src/components/demo/DemoPage.jsx

```diff
@@ -14,15 +14,16 @@ import { DEFAULT_AD } from "./constants";
  * We don't do anything with {matches, path, url} here - we just don't want them in ...props
  * We pull both capitalisations of "nosocial" so we can type the wrong one and still have it work
  */
-const DemoPage = ({device, format, social, matches, path, url, noSocial, nosocial, 'gl.vert': vertId = DEFAULT_AD, ...props}) => {
+const DemoPage = ({device, format, social, matches, path, url, noSocial, nosocial, hide, 'gl.vert': vertId = DEFAULT_AD, ...props}) => {
 	// Allow any noSocial param at all, even an empty one, to work (unless value is "false")
 	const noSocialMerged = (noSocial !== 'false' && noSocial !== undefined) || (nosocial !== 'false' && nosocial !== undefined);
+	const toHide = hide.split(",");
```

Comment:
`hide` comes in as a `string` then `hide` goes out as a `string[]`. If the type changes, then its best to change the variable name. I often add a -s for arrays, e.g. `hides`.